kite-data/kite-data-core/pom.xml

```diff
@@ -271,5 +271,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-deploy-plugin</artifactId>
+            <version>${vers.maven-deploy-plugin}</version>
+            <executions>
+              <execution>
+                <id>deploy-hdp-classifier</id>
```

Comment:
Ah, found this [here](https://maven.apache.org/guides/mini/guide-configuring-plugins.html):
> Note that while execution id's have to be unique among all executions of a single plugin within a POM, they don't have to be unique across an inheritance hierarchy of POMs. Executions of the same id from different POMs are merged. The same applies to executions that are defined by profiles.